### PR TITLE
Added Security to project.

### PR DESCRIPTION
**Pull request by opopovych**
Added roles and authorities.

### CODE REVIEW

File: src/main/java/mate/academy/dto/CreateBookRequestDto.java [hunk 1]
```diff
@@ -16,7 +15,6 @@ public class CreateBookRequestDto {
     @Min(0)
     private BigDecimal price;
     @NotNull
-    @ISBN(type = ISBN.Type.ANY)
     private String isbn;
```
Comment (MentorMA) [inline src/main/java/mate/academy/dto/CreateBookRequestDto.java]: Why you clear isbn anno?

File: src/main/java/mate/academy/model/Role.java [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.Data;
+import org.springframework.security.core.GrantedAuthority;
+
+@Entity
+@Data
+@Table(name = "roles")
+public class Role implements GrantedAuthority {
+
```
Comment (MentorMA) [inline src/main/java/mate/academy/model/Role.java]:
```suggestion
```

File: src/main/resources/application.properties [hunk 3]
```diff
@@ -5,3 +5,4 @@ spring.jpa.show-sql=true
 spring.jpa.hibernate.ddl-auto=validate
 
 
+
```
Comment (MentorMA) [inline src/main/resources/application.properties:8]: For what you add an empty line?

**Review by MentorMA — CHANGES_REQUESTED** [review]

File: src/main/resources/application.properties (hunk 3 above)
Comment (opopovych) [reply src/main/resources/application.properties:8]: removed

File: src/main/java/mate/academy/dto/CreateBookRequestDto.java (hunk 1 above)
Comment (opopovych) [reply src/main/java/mate/academy/dto/CreateBookRequestDto.java]: accidentally

**Pull request merged**